lib/msal-common/src/client/RefreshTokenClient.ts

```diff
@@ -150,7 +151,19 @@ export class RefreshTokenClient extends BaseClient {
         };
 
         const endpoint = UrlString.appendQueryString(authority.tokenEndpoint, queryParameters);
-        return this.executePostToTokenEndpoint(endpoint, requestBody, headers, thumbprint);
+        return this.executePostToTokenEndpoint(endpoint, requestBody, headers, thumbprint)
```

Comment:
Could you reformat this to use "await"?

Reply:
I think using `.then` and `.catch` is fine here, functionality equivalent.

Reply:
That's fine. I was thinking in terms of syntactic sugar.